beacon_node/network/src/sync/block_lookups/tests.rs

```diff
@@ -1232,6 +1388,21 @@ mod deneb_only {
             })
         }
 
+        fn log(self, msg: &str) -> Self {
+            self.rig.log(msg);
+            self
+        }
+
+        // TODO: Eventually deprecate this function
+        fn set_block_id_for_import(mut self) -> Self {
```

Comment:
when should this be deprecated? I'm going to remove the TODO because it should be identified as dead code when we will want to remove it

Reply:
removed in https://github.com/dapplion/lighthouse/pull/26/

Reply:
The deneb test rig made some assumptions about which is the target block to download that somehow broke when modifying the test cases, I think when adding `block_missing_components` function. I could not reason why, so I added this method to get the tests to pass. Seems like a bit of a hack, could you look into it?

Reply:
it makes sense that we wouldn't have a block req id because the block was included in unknown parent error, so no request was ever triggered. i added a fallback to the blob req id here:

4726cbd9819dc5b0484d23d82514637baa2ab4ef

Also seems like there was a duplicate blobs response in the test, which didn't look like what it was trying to test so i removed the second